docs/docs/guide/02-hello.md

```diff
@@ -5,26 +5,23 @@ description: Step-by-step guidance to build your first blockchain and your first
 
 # Hello, Ignite CLI
 
-This tutorial is a great place to start your journey into the Cosmos ecosystem. Instead of wondering how to build a
-blockchain, follow these steps to build your first blockchain and your first Cosmos SDK module.
+This tutorial is a great place to start your journey into the Cosmos ecosystem. Instead of wondering how to build a blockchain, follow these steps to build your first blockchain and your first Cosmos SDK module.
 
 ## Get started
 
-In the previous chapter you've learned how to install [Ignite CLI](https://github.com/ignite/cli), the tool that offers
-everything you need to build, test, and launch your blockchain with a decentralized worldwide community.
+In the previous chapter you've learned how to install [Ignite CLI](https://github.com/ignite/cli), the tool that offers everything you need to build, test, and launch your blockchain with a decentralized worldwide community.
+
+This tutorial is based on a specific version of Ignite CLI, so be sure to install the correct version. 
 
-This series of tutorials is based on the latest version of Ignite CLI, so be sure to install the correct version. 
 Use the following command:
 
 ```bash
-curl https://get.ignite.com/cli! | bash
+curl https://get.ignite.com/cli@v0.25.1! | bash
```

Comment:
With @aljo242, we precisely decided to remove that kind of constraints from our tutorials. See related issue #2902.

Is there any reason why we should stick to this particular version ?

Reply:
I don't think it's possible to just say "the latest version" unless we are committed to updating tutorials to the latest version. Which has not happened reliably in the past.

Reply:
We are planning to have integration tests against all of our tutorials.  I think this should be our goal now anyways.

Reply:
> Which has not happened reliably in the past.

OK I didn't know that.

> We are planning to have integration tests against all of our tutorials. 

That would be nice, there's already an issue for that #2593. I would add that the development of such a feature represents a significant amount of work, and we are not sure that it is feasible.

Reply:
Integration test for tutorials is a great idea! Even if we target the latest stable version, having it in the install script might make it easier for those browsing older versions of tutorials to know which version they need to install.

Reply:
Yeah, that's a good point

Reply:
I think fine to keep this until we have fully implemented tutorial integration tests.